examples/PlotPointwiseGFRE.ipynb

```diff
@@ -0,0 +1,212 @@
+    "            power_spectrum_features, bispectrum_features, train_idx=train_idx, test_idx=test_idx)\n",
+    "\n",
+    "# pointwise global reconstruction error of bispectrum features using power spectrum features mapped to the RKHS\n",
+    "power_spectrum_rbf_to_bispectrum_pointwise_gfre = pointwise_global_reconstruction_error(\n",
+    "            rbf_power_spectrum_features, bispectrum_features, train_idx=train_idx, test_idx=test_idx)\n",
+    "print(\"Computing pointwise GFRE finished.\")\n",
+    "\n",
+    "print(\"Computing GFRE...\")\n",
+    "# global reconstruction error of bispectrum features using power spectrum features\n",
+    "power_spectrum_to_bispectrum_gfre = global_reconstruction_error(\n",
+    "            power_spectrum_features, bispectrum_features, train_idx=train_idx, test_idx=test_idx)\n",
+    "\n",
+    "# global reconstruction error of bispectrum features using power spectrum features mapped to the RKHS\n",
+    "power_spectrum_rbf_to_bispectrum_gfre = global_reconstruction_error(\n",
+    "            rbf_power_spectrum_features, bispectrum_features, train_idx=train_idx, test_idx=test_idx)\n",
+    "print(\"Computing GFRE finished.\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "fig, axes = plt.subplots(1,1, figsize=(12,7))\n",
+    "\n",
+    "bins = np.linspace(0, 0.5, 10)\n",
+    "axes.hist(power_spectrum_to_bispectrum_pointwise_gfre, bins, alpha=0.5, label=\"pointwise GFRE(3-body, 4-body)\")\n",
+    "axes.hist(power_spectrum_rbf_to_bispectrum_pointwise_gfre, bins, color='r', alpha=0.5, label=\"pointwise GFRE(3-body RBF, 4-body)\")\n",
+    "axes.axvline(power_spectrum_to_bispectrum_gfre, color='darkblue', label=\"GFRE(3-body, 4-body)\", linewidth=4)\n",
+    "axes.axvline(power_spectrum_rbf_to_bispectrum_gfre, color='darkred', label=\"GFRE(3-body RBF RKHS, 4-body)\", linewidth=4)\n",
+    "axes.set_title(f\"3-body vs 4-body RBF gamma={gamma} comparison\")\n",
+    "axes.set_xlabel(\"pointwise GFRE\")\n",
+    "axes.set_ylabel(\"number of samples\")\n",
+    "axes.legend(fontsize=13)\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(\"GFRE(3-body, 4-body) =\", power_spectrum_to_bispectrum_gfre)\n",
+    "print(\"GFRE(3-body RBF RKHS, 4-body) = \", power_spectrum_rbf_to_bispectrum_gfre)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "It can be seen that RBF RKHS features improve the linear reconstruction of the 4-body features (~0.22 in contrast to ~0.19) while also spreading the error for individual samples across a wider span of [0, 0.5] in contrast to [0.17, 0.32]. This indicates that the reconstruction using the RBF RKHS is more particular to the dataset, and is therefore less general."
```

Comment:
```suggestion
    "It can be seen that RBF RKHS features improve the linear reconstruction of the 4-body features (~0.22 in contrast to ~0.19) while also spreading the error for individual samples across a wider span of [0, 0.5] in contrast to [0.17, 0.32]. This indicates that the reconstruction using the RBF RKHS is less generally applicable but instead specific to this dataset."
```

Reply:
changed